### DISCUSSION TITLE
Directory Connector v2.9.3 - No longer working

### DISCUSSION THREAD

**Discussion by jakeman2048** [dani-garcia/vaultwarden#1875] [Ideas]
It seems the [newest version of directory connector](https://github.com/bitwarden/directory-connector/releases/tag/v2.9.3) no longer works with vaultwarden, as it now uses the API endpoints. Are there any future plans to get this working?

**Comment by BlackDex** [comment]
That feature isn't supported yet indeed.
You can always use https://github.com/bitwarden/directory-connector/releases/tag/v2.9.2 to have it still working for now.
